### PR TITLE
Review PR for BXMSDOC-6601 Update getting started with processes doc to match latest UI for 7.9

### PR DESCRIPTION
**Pull request by michelehaglund**
Original jira: https://issues.redhat.com/browse/BXMSDOC-6601
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-6601/#assembly-getting-started-process-services (Chapter 7 - 19)

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc [hunk 1]
```diff
@@ -22,15 +22,17 @@ NOTE: Red Hat does not provide support for the sample code included in the {PROD
 
 The following sample projects are available in {CENTRAL}:
 
+* *Course_Scheduling*: (Business optimization) Course scheduling and curriculum decision process. Assigns lectures to rooms and determines a student's curriculum based on factors, such as course conflicts and class room capacity.
```
Comment (emmurphy1) [inline doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc:25]: factors such as (remove comma)

File: doc-content/enterprise-only/getting-started/process-instances-filters-advanced-proc.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ Use the *Advanced Filters* option to create custom process instance filters. The
 
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
-. On the *Manage Process Instances* page, click *Advanced Filters* icon.
+. On the *Manage Process Instances* page, click *Advanced Filters* icon on the left of the page.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/process-instances-filters-advanced-proc.adoc:7]: I would reword:
On the *Manage Process Instances* page, on the left of the page click the  *Advanced Filters* icon .
or
On the *Manage Process Instances* page, on the left of the page click  *Advanced Filters* .
 .

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [id='process-instances-filters-proc']
 = Filtering process instances using default filters
-Filter processes instances by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*. 
+Filter processes instances by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:3]: Filter process instances?

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc [hunk 4]
```diff
@@ -17,8 +17,10 @@ This pane lists the following process attributes which you can use to filter pro
 ... Click *Apply*.
 +
 
-* *Name*: Filter process instances based on process definition name.
-* *Start Date*: Filter process instances based on their creation date.
-* *Last update*: Filter process instances based on their last modified date.
-
-You can also use the *Advanced Filters* option to create custom filters in {CENTRAL}.
+* *Name*: Filter process instances by definition names.
+* *Definition Id*: Filter process instances by process definition Ids.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:21]: Normally, I'd say IDs not Ids. Search in doc. But I see it's Id in the UI. Ugh...

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc [hunk 5]
```diff
@@ -17,8 +17,10 @@ This pane lists the following process attributes which you can use to filter pro
...
+* *Deployment Id*: Filter process instances by process deployment Ids.
+* *Parent Process Instance Id*: Filter process instances by parent process instance Ids.
+* *SLA Compliance*: Filter process instances by SLA Compliance states.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:24]: SLA compliance  states.

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc:25]: Done

File: doc-content/enterprise-only/getting-started/process-instances-filters-advanced-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/process-instances-filters-advanced-proc.adoc:7]: Done

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc (hunk 3 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:3]: Ooopsie. Fixed.

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc (hunk 4 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:21]: I changed the instances that were not UI replicas

File: doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc (hunk 5 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/process-instances-filters-proc.adoc:24]: Fixed.

**Comment by LuboTerifaj** [comment]
It looks good to me.
